frontend/src/modules/filters/organizer/adapter.ts

```diff
@@ -0,0 +1,18 @@
+import { ORGANIZER_ID } from '../constant';
+import { FilterWithoutType } from '../interface';
+import { RawOrganizer } from './interface';
+
+export const adaptOrganizerFilter = (
+  rawOrganizers: RawOrganizer[] | null,
+): FilterWithoutType | null => {
+  if (rawOrganizers === null) {
+    return null;
+  }
+  return {
+    id: ORGANIZER_ID,
+    options: rawOrganizers.map(rawOrganizer => ({
+      value: `${rawOrganizer.id}`,
```

Comment:
Why the template string?

Reply:
All API IDs are given as `number`, and the APP uses them as `string`.
For example: when filtering search results, filters are sorted alphabetically https://github.com/GeotrekCE/Geotrek-rando-v3/blob/2b7b0714a4e0933f36619096a7c5710473d2053e/frontend/src/components/pages/search/components/useFilters.ts#L64

NB: To be more explicit, I could use `String()` or `.toString()` but it needed to change for all filters module to be consistent.